config/peerpods/podvm/libvirt-podvm-image-handler.sh

```diff
@@ -158,6 +163,183 @@ function download_rhel_kvm_guest_qcow2() {
 
 }
 
+# Function to extract RVPS value from PODVM SE image
+function calculate_rvps_qcow2() {
+    mkdir -p "${CAA_SRC_DIR}"/podvm/rvps_config
+    cp "${CAA_SRC_DIR}"/podvm/output/*.qcow2 "${CAA_SRC_DIR}"/podvm/rvps_config/podvm-libvirt.qcow2
+    export SE_IMG_PATH="${CAA_SRC_DIR}"/podvm/rvps_config/podvm-libvirt.qcow2
+    export RVPS_CONFIG_DIR="${CAA_SRC_DIR}"/podvm/rvps_config
```

Comment:
do we need to export these as we are not using them beyond the scope of this function?

Reply:
Removed the export.